netbout-client/src/test/java/com/netbout/client/RtFriendsITCase.java

```diff
@@ -109,4 +109,18 @@ public void throwsExceptionIfInvalidToken() throws Exception {
         final String friend = "bob";
         bout.friends().invite(friend);
     }
+
+    /**
+     * RtFriends can throw an exception if unknown alias kicked.
+     * @throws Exception If there is some problem inside
+     */
+    @Test (expected = Friends.UnknownAliasException.class)
+    public void throwsExceptionIfUnknownAliasKicked() throws Exception {
+        final Inbox inbox = this.rule.get().aliases()
```

Comment:
@dmzaytsev All variables here are still single-use variables, we can inline everything (i.e. make the whole test a single statement).